src/Configuration/PHPUnitConfig.php

```diff
@@ -26,6 +26,16 @@ public function __construct(string $inputPathOrFileName)
         $this->xmlLoader = new Loader();
     }
 
+    public function getRootAttribute(string $name): ?string
```

Comment:
I wasn't sure about what this does. Maybe renaming like this make it clearer?
```suggestion
    public function getRootAttributeValue(string $name): ?string
```